src/components/Search.jsx

```diff
@@ -84,6 +84,51 @@ class Search extends React.Component {
             </div>
           </form>
         </div>
+        {/* mobile viewer */}
+        <div className="row hide-on-med-and-up">
+        <form onSubmit={this.handleSubmit} className="col s12 card" style={{padding: 20}}>
+          <h5 className="">Find a restaurant</h5>
+            <div className="row">
+              <div className="input-field col s12 m8">
+                <i className="material-icons prefix">restaurant</i>
+                <label>Restaurant Name</label>
+                <input
+                  type="text"
+                  name="restaurant"
+                  placeholder="Restaurant name"
+                  ref={this.dbaInput}        
+                  required
+                />
+              </div>
+              {/* borough dropdown */}
+              <div className="input-field col s12 m4">
+                <label>Borough</label>
+                <i className="material-icons prefix">location_city</i>
+                <select
+                  name="boro"
+                  ref={this.boroInput}
+                  id="select-override"
```

Comment:
Two `<select>` menus share the same id, `select-override`, on lines [68](https://github.com/CodeWritingCow/NYC-restaurant-inspection-records/blob/dceb30648449ffcec768eedfae48be44eeb70293/src/components/Search.jsx#L68) and [110](https://github.com/CodeWritingCow/NYC-restaurant-inspection-records/blob/dceb30648449ffcec768eedfae48be44eeb70293/src/components/Search.jsx#L110). An id belongs to one element only. Consider changing `select-override` to a class in `<Search>` and [style.css](https://github.com/omenwolf/NYC-restaurant-inspection-records/blob/c8ed0ce3d44d79d5575d2e4d266a984e6ccd505a/dist/style.css#L44)